azure-cli/monitor/get_ApplicationInsights_ConnectionString.ps1

```diff
@@ -0,0 +1,21 @@
+function Get-ApplicationInsightsConnectionString {
```

Comment:
I don't know where the habit of using camel case and underscores for filenames in this project came from, and I haven't seen any other PowerShell projects doing that either.

Going forward, the file names should be identical to the functions, just with a `.ps1` extension

Reply:
Renamed